--- cmake/3rdParty/BuiltInPackages.cmake
+++ cmake/3rdParty/BuiltInPackages.cmake
@@ -19,12 +19,3 @@ include(${LY_PAL_PACKAGE_FILE_NAME})
 
 # add the above file to the ALLFILES list, so that they show up in IDEs
 set(ALLFILES ${ALLFILES} ${LY_PAL_PACKAGE_FILE_NAME})
-
-# temporary compatibility: 
-# Some 3p libraries may still refer to zlib as "3rdParty::zlib" instead of
-# the correct "3rdParty::ZLIB" (Case difference).  Until those libraries are updated
-# we alias the casing here.  This also provides backward compatibility for Gems that use 3rdParty::zlib
-# that are not part of the core O3DE repo.
-ly_download_associated_package(ZLIB)
-find_package(ZLIB)
-add_library(3rdParty::zlib ALIAS 3rdParty::ZLIB)
