--- cmake/Platform/Linux/Configurations_linux.cmake
+++ cmake/Platform/Linux/Configurations_linux.cmake
@@ -42,11 +42,13 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
                 -msse4.1
             LINK_NON_STATIC
                 -Wl,--no-undefined
+                -fuse-ld=lld
                 -fpie
                 -Wl,-z,relro,-z,now
                 -Wl,-z,noexecstack
             LINK_EXE
                 -fpie
+                -fuse-ld=lld
                 -Wl,-z,relro,-z,now
                 -Wl,-z,noexecstack
         )
